### PR TITLE
rpolars_style + utils

### PR DESCRIPTION
**Pull request by sorhawell**

### CODE REVIEW

**Review by eitsupi — CHANGES_REQUESTED** [review]
Thanks for working on this!
I'm sorry, but I'm on a business trip and can't see this in detail for a while.

For the time being, since the tools directory is for storing files that are used when installing R packages, could you please move this to the dev directory?

**Comment by sorhawell** [comment]
I will try to incorporate this in the current Makefile as a suggestion of how it could be used. I know there might be an upcoming Tasks file to use instead

**Comment by sorhawell** [comment]
besides running like this to style changes, it takes 1-8 seconds
```sh
make fmt-r
```

sometimes every .e.g. .R file could be styled something like like this, it would probably take 1-2 min
```R
source("dev/styler_utils.R")
style_files(list.files(".",full.names=TRUE, pattern = "*.R$"))
```

or using the styler package directly. This could take 2-10 min
```R
source("dev/styler_utils.R")
styler::style_pkg(".", style = rpolars_style)
```

**Comment by sorhawell** [comment]
@eitsupi  I don't know why but it seems check is disabled for some reasone

Styling is now fairly verbose, we can set `verbose = FALSE`

File: dev/styler_utils.R [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    pattern = "\\.R$|\\.Rmd$",
+    # specific files never to style
+    excludes = c("R/extendr-wrappers.R")) {
+  # temporarily set GIT_DIF_TARGET
+  old_env = Sys.getenv("GIT_DIF_TARGET")
+  on.exit({
+    Sys.setenv("GIT_DIF_TARGET" = old_env)
+  })
+  if (is.null(branch)) {
+    Sys.setenv("GIT_DIF_TARGET" = "main")
```
Comment (sorhawell) [inline dev/styler_utils.R:68]: Should touch GIT_DIF_TARGET if branch is NULL

**Review by eitsupi — APPROVED** [review]
Seems working fine, thanks!

**Comment by eitsupi** [comment]
@sorhawell If we use this, it seems that `<<-` will be changed to `=`. Could you fix it?

**Comment by eitsupi** [comment]
@sorhawell Could you fix this?

**Pull request merged**